Gruntfile.js

```diff
@@ -22,13 +22,16 @@ module.exports = function (grunt) {
                 options: {
                     reporter: "spec"
                 },
-                src: ["build/test/**/*.js"]
+                src: ["build/test/**/*Tests.js", "build/test/assert.js", "build/test/lint.js"]
             }
         },
 
         run: {
-            test: {
-                cmd: "./test/check-bin.sh"
+            testBin: {
+                exec: "./test/check-bin.sh"
+            },
+            testRules: {
+                exec: "node ./build/test/ruleTestRunner.js"
```

Comment:
looking at the docs for grunt-run, I think simply `args` is preferable here to `exec`. `node` is the default command used by grunt-run, so this can just be `args: [ "build/test/ruleTestRunner.js" ]`